tests/test_etherscan.py

```diff
@@ -0,0 +1,17 @@
+import json
+
+import pytest
+from requests import Response
+
+
+@pytest.fixture
+def etherscan_abi_response(mocker):
+    response = mocker.MagicMock(spec=Response)
+    f = open("api.json")
+    response.json.return_value = json.load(f)
```

Comment:
We need to make sure we close the file after this `f.close()`

Reply:
fixed in bbb6a81